scapy/contrib/scada/iec104/iec104.py

```diff
@@ -58,6 +58,9 @@
 from scapy.packet import Raw
 from scapy.packet import Packet, bind_layers
 
+from . import iec104, iec104_fields, iec104_information_elements, \
+    iec104_information_objects  # noqa: F401
```

Comment:
This fixes some import errors, due to the fact that this file has the same name than its module.